releases.md

```diff
@@ -40,6 +40,13 @@ Further documentation available:
 
 ## Releases
 
+### v0.28
+
+- **Latest Release**: [v0.28.0][v0-28-0] (2024-07-08) ([docs][v0-28-0-docs], [examples][v0-28-0-examples])
+- **Initial Release**: [v0.28.0][v0-28-0] (2024-07-08)
```

Comment:
@khrm what is the dates for latest and initial release and what is the use of these dates ?

Reply:
Patch release will change the initial dates.